ui/assets/js/common.js

```diff
@@ -5,7 +5,7 @@ var lib = {
 
     ExpandVars: function(template, values) {
       for (var name in values) {
-        template = template.replace('{' + name + '}', values[name]);
+        template = template.replace(new RegExp('{' + name + '}', 'g'), values[name]);
```

Comment:
Bitbucket's URL specifies the branch twice; this handles replacing multiple instances.